### PR TITLE
Adding mf2 package, v2020.4.3

### PR DESCRIPTION
**Pull request by sjvrijn**
Hi python team, my PR is ready for review/inclusion 

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by sjvrijn** [comment]
@conda-forge-admin, please ping team

File: recipes/mf2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - numpy
+    - python
+
+test:
+  requires:
+    - setuptools
+  imports:
+    - mf2
+    - mf2.adjustable
+    - tests
```
Comment (marcelotrevisani) [inline recipes/mf2/meta.yaml:33]:
```suggestion
```

File: recipes/mf2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+about:
+  home: https://github.com/sjvrijn/mf2
+  license: GNU General Public v3 (GPLv3)
+  license_family: GPL3
```
Comment (marcelotrevisani) [inline recipes/mf2/meta.yaml:38]:
```suggestion
  license_family: GPL
```

File: recipes/mf2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy
+    - pip
+    - python
+  run:
+    - setuptools
```
Comment (marcelotrevisani) [inline recipes/mf2/meta.yaml:23]: do you really need ``setuptools`` to run this project?
```suggestion
```

File: recipes/mf2/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy
+    - pip
+    - python
```
Comment (marcelotrevisani) [inline recipes/mf2/meta.yaml:21]:
```suggestion
    - python >=3.6
```

File: recipes/mf2/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  sha256: 43e0be769160e19459d6cf7adb98373f8d6042bf4ca804049bc1aba58333851c
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - numpy
```
Comment (marcelotrevisani) [inline recipes/mf2/meta.yaml:19]:
```suggestion
```

File: recipes/mf2/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+  requires:
+    - setuptools
+  imports:
+    - mf2
+    - mf2.adjustable
+    - tests
+
+about:
+  home: https://github.com/sjvrijn/mf2
+  license: GNU General Public v3 (GPLv3)
```
Comment (marcelotrevisani) [inline recipes/mf2/meta.yaml:37]:
```suggestion
  license: GPL-3.0-or-later
```

File: recipes/mf2/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 43e0be769160e19459d6cf7adb98373f8d6042bf4ca804049bc1aba58333851c
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/mf2/meta.yaml:15]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

File: recipes/mf2/meta.yaml [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+
+requirements:
+  host:
+    - numpy
+    - pip
+    - python
+  run:
+    - setuptools
+    - numpy
+    - python
```
Comment (marcelotrevisani) [inline recipes/mf2/meta.yaml:25]:
```suggestion
    - python >=3.6
```

**Comment by marcelotrevisani** [comment]
If you want to, next time you can give a chance to [Grayskull :skull: ](https://github.com/marcelotrevisani/grayskull/) to generate the recipes for PyPI packages 
:)

File: recipes/mf2/meta.yaml (hunk 3 above)
Comment (sjvrijn) [reply recipes/mf2/meta.yaml:23]: I'm defining the version number only in `setup.py`, and then importing it in `__init__.py` using `pkg_resources` from `setuptools`.
See https://github.com/sjvrijn/mf2/blob/master/mf2/__init__.py#L28

**Comment by sjvrijn** [comment]
@marcelotrevisani Thanks for taking the time to review this recipe. I'm not yet very familiar with conda recipes, so I'll definitely look into your Grayskull suggestion 👍 
In any case, I've accepted all your remaning suggested changes.

**Pull request merged**